src/common/transformations/src/transformations/fp16_compression/mark_subgraphs_to_keep_in_mixed_precision.cpp

```diff
@@ -209,6 +227,58 @@ class InitMarkReduceOpPath : public pass::MatcherPass {
     }
 };
 
+/* MarkFloatingPointRange marks paths that involve Range operations with floating point output data types,
+ * as well as their users allowed for propagation. This pass is needed to prevent accuracy data loss
+ * in cases of high range generation, which could suffer due to lowered precision.
+ */
+class MarkFloatingPointRange : public pass::MatcherPass {
+public:
+    OPENVINO_RTTI("MarkFloatingPointRange", "0");
+    MarkFloatingPointRange() {
+        MATCHER_SCOPE(MarkFloatingPointRange);
+        auto exp_pattern = pattern::wrap_type<ov::op::v4::Range>();
+
+        // through this nodes
+        const std::shared_ptr<Node> range_propagating_nodes = pattern::wrap_type<ov::op::v0::Convert,
```

Comment:
should we connect `exp_pattern` (Range) and `range_propagating_nodes` nodes?
I don't see the connection Range -> (Convert, Greater etc) 
To add this connection, you need to pass `exp_pattern` to the constructor of range_propagating_nodes wrap_type

minor:
I believe you can use auto here: `const auto range_propagating_nodes`

Reply:
Oops, it was a copy-paste mistake, this `exp_pattern` was not used at all. I removed it, thanks